packages/migration-graph/tests/fixtures/project.ts

```diff
@@ -0,0 +1,109 @@
+import { Project } from 'fixturify-project';
+import { dirname, join } from 'path';
+import merge from 'lodash.merge';
+import tmp from 'tmp';
+import { execSync } from 'child_process';
+import { existsSync } from 'fs-extra';
+
+tmp.setGracefulCleanup();
```

Comment:
minor - but can we fix the ordering here with all the imports above the invocations. eg `tmp.setGracefulCleanup();` moved below the final import